configs/testing/daal4py_xgboost.json

```diff
@@ -0,0 +1,32 @@
+{

```

Comment:
What do you think about renaming it to `daal4py_xgboost_interop.json`? Is it applicable?